test/frontend/tests/workflow-tests.js

```diff
@@ -18,87 +20,61 @@ navigator.serviceWorker.register('/serviceWorker.js');
 
 describe('Upload / Download flow', function() {
   this.timeout(0);
-  it('can only download once by default', async function() {
```

Comment:
Can we keep the original logic? It should throw an error when download was performed previously

Reply:
Our default behavior would be to keep files until they are deleted. But yes, good point, thanks, restored.